playkit/src/main/java/com/kaltura/playkit/player/SubtitleStyleSettings.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * ============================================================================
+ * Copyright (C) 2018 Kaltura Inc.
+ *
+ * Licensed under the AGPLv3 license, unless a different license for a
+ * particular library is specified in the applicable library path.
+ *
+ * You may obtain a copy of the License at
+ * https://www.gnu.org/licenses/agpl-3.0.html
+ * ============================================================================
+ */
+
+package com.kaltura.playkit.player;
+
+import android.graphics.Color;
+import android.graphics.Typeface;
+import com.google.android.exoplayer2.text.CaptionStyleCompat;
+
+public class SubtitleStyleSettings {
```

Comment:
Create toCaptionStyle() method that will create and return a CaptionStyleCompat object.